localization/english/event_localization/wc_major_decisions_events_l_english.yml

```diff
@@ -0,0 +1,41 @@
+ wc_holy_order_decision.0011.desc.northshire_stay:0 "[founder.GetTitledFirstName] has sent word to all priests of [founder.GetFaith.HighGodName]. War is coming, and the faithful must be ready.\n\nIf humanity and its allies are to survive the coming era of strife, it will need warriors who can heal the innocent and strike down the wicked.\n\nThese warriors will be called "paladins". Five men have already been chosen as the first of them. Famed for their strength of arm and devotion, each has been granted a libram of power and tasked with training their juniors.\n\nIn ancient times, humanity was said to have a protector, the mythical hero Tyr. This order of paladins will be known as the Knights of [leader.GetPrimaryTitle.GetBaseName], named in honor of this legendary figure."
+ wc_holy_order_decision.0011.desc.original_paladin:0 "[founder.GetTitledFirstName] has sent word to all priests of [founder.GetFaith.HighGodName]. War is coming, and the faithful must be ready.\n\nIf humanity and its allies are to survive the coming era of strife, it will need warriors who can heal the innocent and strike down the wicked.\n\nThese warriors will be called "paladins". Among the faithful, five men, including myself, have been chosen as the inaugural warriors. Renowned for our strength and devotion, a libram of power has been bestowed upon each of us, entrusting us with the duty to train those who will follow. Since ancient times, it is said that humanity had a protector, the mythical hero Tyr. Our order of paladins will be known as the Knights of [leader.GetPrimaryTitle.GetBaseName], in honor of this legendary figure."
+ wc_holy_order_decision.0011.desc.original_paladin_northshire_stay:0 "[founder.GetTitledFirstName] has sent word to all priests of [founder.GetFaith.HighGodName]. War is coming, and the faithful must be ready.\n\nIf humanity and its allies are to survive the coming era of strife, it will need warriors who can heal the innocent and strike down the wicked.\n\nThese warriors will be called "paladins". Among the faithful, five men, including myself, have been chosen as the inaugural warriors. Renowned for our strength and devotion, a libram of power has been bestowed upon each of us, entrusting us with the duty to train those who will follow. Since ancient times, it is said that humanity had a protector, the mythical hero Tyr. Our order of paladins will be known as the Knights of [leader.GetPrimaryTitle.GetBaseName], in honor of this legendary figure."
```

Comment:
Bro these are all basically the same. Please put the parts that are the same in another loc and then include it using `$<loc key>$`